Formula/ooniprobe.rb

```diff
@@ -169,10 +167,6 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources", "--update-inputs", "--update-geoip"
```

Comment:
I'm not sure in our case, given we bottle things, that doesn't mean we don't want to retain this. Most people will pour the bottles, and for example the bottles are generated today so someone installing in a couple of months would still get the `inputs` and `geoip` from today as-is.